pkg/securityscan/alert/prometheusrule.go

```diff
@@ -0,0 +1,74 @@
+package alert
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/sirupsen/logrus"
+	meta1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	k8Yaml "k8s.io/apimachinery/pkg/util/yaml"
+
+	cisoperatorapiv1 "github.com/rancher/cis-operator/pkg/apis/cis.cattle.io/v1"
+	"github.com/rancher/wrangler/pkg/name"
+)
+
+func NewPrometheusRule(clusterscan *cisoperatorapiv1.ClusterScan, clusterscanprofile *cisoperatorapiv1.ClusterScanProfile, imageConfig *cisoperatorapiv1.ScanImageConfig) (*monitoringv1.PrometheusRule, error) {
+	configdata := map[string]interface{}{
+		"namespace":       cisoperatorapiv1.ClusterScanNS,
+		"name":            name.SafeConcatName("rancher-monitoring-cis-alerts", clusterscan.Name),
+		"severity":        imageConfig.AlertsSeverity,
+		"scanName":        clusterscan.Name,
+		"scanProfileName": clusterscanprofile.Name,
+		"alertOnFailure":  clusterscan.Spec.ScheduledScanConfig.ScanAlertRule.AlertOnFailure,
+		"alertOnComplete": clusterscan.Spec.ScheduledScanConfig.ScanAlertRule.AlertOnComplete,
+	}
+	scanAlertRule, err := generatePrometheusRule(clusterscan, "alertingRule.template", "./pkg/securityscan/alert/templates/alertingRule.template", configdata)
+	if err != nil {
+		return scanAlertRule, err
+	}
+
+	return scanAlertRule, nil
+}
+
+func generatePrometheusRule(clusterscan *cisoperatorapiv1.ClusterScan, templateName string, templateFile string, data map[string]interface{}) (*monitoringv1.PrometheusRule, error) {
+	scanAlertRule := &monitoringv1.PrometheusRule{}
+	logrus.Infof("config %v", data)
```

Comment:
Seems sparse for an Info log. Should this be debug or should this be more verbose?

Reply:
removed, that was for debugging while testing